packages/spark-core/components/_buttons.scss

```diff
@@ -80,12 +80,12 @@
   &:active::after,
   &:focus::after {
     border-color: $btn-tertiary-hover-line-color;
-    animation: width-zero-to-full $btn-transition-speed ease-in-out;
+    transition: border $btn-transition-speed ease-in-out;
   }
```

Comment:
I don't think this transition is coming through. It needs to be set not in the hover, active ETC states, but in default state.

Reply:
The desired behavior is that the underline on tertiary buttons fades from $gray-dark to $black over .3 seconds when hovered.